### PR TITLE
create keras example in flower/flwr_examples

### PR DESCRIPTION
**Pull request by mariaboerner1987**

### CODE REVIEW

File: examples/quickstart_tensorflow/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+```shell
+pyenv virtualenv 3.7.9 keras-federated-3.7.9
+```
+
+Activate the virtualenv by creating a `.python-version` with the content:
+
+```shell
+keras-federated-3.7.9
+```
```
Comment (tanertopal) [inline examples/quickstart_tensorflow/README.md:27]:
```suggestion
```
I would skip this section and remove it as poetry by default creates a virtualenv. I just tried it out with the example and it worked well.

File: examples/quickstart_tensorflow/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# Flower Example using Keras
+
+Flower is easy to use by only starting the Flower server and the Flower clients.
+
+It is recommended to use [pyenv](https://github.com/pyenv/pyenv)/[pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) and [poetry](https://python-poetry.org/docs/) to ensure the right version of libraries.
+
+After installing both tools you can start to set up everything.
```
Comment (tanertopal) [inline examples/quickstart_tensorflow/README.md:7]:
```suggestion
This example uses [poetry](https://python-poetry.org) for dependency management. If not checkout the [poetry](https://python-poetry.org) information howto install it.
```

File: examples/quickstart_tensorflow/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+Activate the virtualenv by creating a `.python-version` with the content:
+
+```shell
+keras-federated-3.7.9
+```
+
+## Run Keras Federated
+
+Clone the flower examples to your virtualenv:
```
Comment (tanertopal) [inline examples/quickstart_tensorflow/README.md:31]:
```suggestion
Start with cloning the Flower repo and checking out the example. We have prepared a single line which you can copy into your shell which will checkout the example for you.
```

File: examples/quickstart_tensorflow/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+./run-server.sh
+```
+
+Open a new terminal and start 2 Flower clients with
+
+```shell
+./run-client.sh
+```
+
+You will see that Keras is starting a federated training. Have a look to the [Flower Quickstarter documentation](https://flower.dev/docs/quickstart_tensorflow.html) for a detailed explanation of the code.
```
Comment (tanertopal) [inline examples/quickstart_tensorflow/README.md:66]:
```suggestion
You will see that Keras is starting a federated training. Have a look to the [Flower Quickstarter documentation](https://flower.dev/docs/quickstart_tensorflow.html) for a detailed explanation.
```

**Review by tanertopal — CHANGES_REQUESTED** [review]

File: examples/quickstart_tensorflow/README.md (hunk 2 above)
Comment (mariaboerner1987) [reply examples/quickstart_tensorflow/README.md:10]: The changes will be added in the next PR

File: examples/quickstart_tensorflow/README.md (hunk 1 above)
Comment (mariaboerner1987) [reply examples/quickstart_tensorflow/README.md:30]: This will be implemented in the next PR.

File: examples/quickstart_tensorflow/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+This introductory example to Flower uses Keras but deep knowledge of Keras is not necessarily required to run the example. However, it will help you understanding how to adapt Flower to your use-cases.
+Running this example in itself is quite easy.
+
+It is recommended to use [pyenv](https://github.com/pyenv/pyenv)/[pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) and [poetry](https://python-poetry.org/docs/) to ensure the right version of libraries.
+
+After installing both tools you can start to set up everything.  
+
+## Setup your virtual environment
+
+First, you should setup your virtualenv and  install [Python Version 3.6](https://docs.python.org/3.6/) or above:
```
Comment (tanertopal) [inline examples/quickstart_tensorflow/README.md:12]:
```suggestion
First, you should setup your virtualenv and install [Python Version 3.6](https://docs.python.org/3.6/) or above:
```

File: examples/quickstart_tensorflow/README.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+pyenv install 3.7.9
+```
+
+Create a virtualenv with:
+
+```shell
+pyenv virtualenv 3.7.9 keras-federated-3.7.9
+```
+
+Activate the virtualenv by creating a `.python-version` with the content:
```
Comment (tanertopal) [inline examples/quickstart_tensorflow/README.md:24]:
```suggestion
Activate the virtualenv by running the following command:
```

File: examples/quickstart_tensorflow/README.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+```shell
+keras-federated-3.7.9
```
Comment (tanertopal) [inline examples/quickstart_tensorflow/README.md:27]:
```suggestion
echo keras-federated-3.7.9 > .python-version
```

**Review by tanertopal — CHANGES_REQUESTED** [review]
Left few suggestions

**Review by tanertopal — APPROVED** [review]

**Pull request merged**